pyOpenSci blogpost (#401)
sunpy/sunpy.org: posts/2024/2024-01-24-pyopensci.md

---
blogpost: true
date: Jan 24 2024
author: Nabil Freij
category: Update
---

# pyOpenSci and `sunpy`

![pyOpenSci logo](https://avatars.githubusercontent.com/u/28938222?s=200&v=4)

[pyOpenSci](https://www.pyopensci.org/) is a "diverse community of people interested in building a community of practice around scientific software written in Python".

> They are an exciting community working on improving the information around the packaging and tooling used by the larger community as well to provide support for package maintainers.

Reviewer: ```suggestion
They are an exciting community working on improving the information around the packaging and tooling used by the scientific Python community as well to provide support for package maintainers.
```

> There is a review process to become a pyOpenSci accepted package which comes with two extensive reviews from independent members of the wider community.

Reviewer: ```suggestion
There is a review process to become a pyOpenSci accepted package which comes with two reviews from independent members of the wider community.
```

> With this in mind, the SunPy Project decided to submit `sunpy` to pyOpenSci to ensure that it is aligned with the broader Python community that pyOpenSci is building.

Reviewer: ```suggestion
With this in mind, the SunPy Project decided to submit the `sunpy` package to pyOpenSci to ensure that it is aligned with the community that pyOpenSci is building, and increase visibility of SunPy.
```

> The SunPy Project strives to be a useful member of the Scientiﬁc Python ecosystem and this is a good opportunity.

Reviewer: This sentence sounds very disconnected.
```suggestion
```

[The review](https://github.com/pyOpenSci/software-submission/issues/147) was started on the [pyOpenSci software-submission repository](https://github.com/pyOpenSci/software-submission).
The entire review process is open, with one editor and two reviewers and the entire exchange between the `sunpy` maintainers and the reviewers are on that issue.

> This process was incredibly helpful as it pointed some miscommunication within our documentation and some choices that were made several years ago but now were redundant or not best practice.

Reviewer: ```suggestion
This process was incredibly helpful as it highlighted areas of our documentation which could be improved and some technical choices that were made several years ago but now were redundant or not best practice.
```
